docs/sdk/authentication/session-sigs/get-lit-action-session-sigs.md

```diff
@@ -0,0 +1,168 @@
+---
+sidebar_position: 2
+---
+
+import FeedbackComponent from "@site/src/pages/feedback.md";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Generating SessionSigs: `getLitActionSessionSigs`
+
+This guide covers the `getLitActionSessionSigs` function from the Lit SDK. For an overview of what session signatures are and how they are to be used, please go [here](./intro).
+
+Using the `getLitActionSessionSigs` function, you can specify the capabilities of your current session on the Lit network. 
+
+This function is very similar to [`getPkpSessionSigs`](./get-pkp-session-sigs.md). The `getPkpSessionSigs` function requires you to own a PKP and some form of authentication to prove your identity (e.g. a custom Lit Action, AuthMethod, or AuthSig).
```

Comment:
it could be helpful to add a few more sentences here that clearly articulate the differences between these two functions and why one would use one over the other

Reply:
I think this is the only major difference? Most of the `getLitActionSessionSig` properties are inherited from `getPkpSessionSigs`. Their generated session signatures are identical as well:

> This function is very similar to [`getPkpSessionSigs`](./get-pkp-session-sigs.md). The `getPkpSessionSigs` function requires you to own a PKP and some form of authentication to prove your identity (e.g. a custom Lit Action, AuthMethod, or AuthSig).
> 
> Alternatively, the `getLitActionSessionSigs` function requires the form of authentication to be a Lit Action. Other authentication methods can be included as well, but a Lit Action is required.